src/Tizen.NUI.Wearable/src/public/WearableStyle/PopupStyle.cs

```diff
@@ -0,0 +1,91 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+using Tizen.NUI.Components;
+
+namespace Tizen.NUI.Wearable
+{
+    /// <summary>
+    /// PopupStyle used to config the Popup represent.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class PopupStyle : ControlStyle
+    {
+        static PopupStyle() { }
+
+        /// <summary>
+        /// Creates a new instance of a PopupStyle.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public PopupStyle() : base()
+        {
+            initSubStyle();
```

Comment:
What's the meaning of SubStyle?

Reply:
I followed other Style class naming examples. I think it's because popup style inherits from control style. so it means child class's initializing, I think...